### ISSUE TITLE
This repo helped me a lot, I am curious whether it was coded by you?

### ISSUE THREAD

**Issue by BokyLiu** [jizhuoran/caffe-huawei-atlas-convertor#1]

**Comment by jizhuoran** [comment]
Yes. New feature needed?

**Comment by BokyLiu** [comment]
> Yes. New feature needed?

yes, need a lot. Clamp, InstanceNorm and etc.
So, if you are working in huawei, why don't consider support onnx to om model

**Comment by jizhuoran** [comment]
I am not working Huawei. May I know you device model? Actually, if you use edge device (e.g., Atlas 200), using CPU to do normalisation is faster than using AI core.

**Comment by BokyLiu** [comment]
Yes, Atlas 200.
How can i use cpu to do normalisation, by sperating model to several parts?
e.g.:
module before normalize    on AI core
normalize                            on cpu
module after normalize       on AI core

**Comment by jizhuoran** [comment]
Yes. I am not sure whether you have some experience with Caffe, it's easy to use different backend for each layer. (BTW, Caffe's CPU implementation is not that Good, we are developing our own one, which is hundreds faster...)

**Comment by BokyLiu** [comment]
yes, i see your hyperTea repo. But in my case, i think it is unwise to cut off my model to dozens of sub models...
Thanks a lot!
